Gemfile

```diff
@@ -53,6 +53,8 @@ gem 'geocoder'
 # Request throttling/blocking management
 gem 'rack-attack'
 
+# Use Uglifier as compressor for JavaScript assets
+gem 'uglifier'
```

Comment:
Added this Gem because I was getting this error `LoadError: cannot load such file -- uglifier`. But still getting the same error. Any help would be appreciated. Thanks